k8s cron job for performance tests (#1030)
kyma-project/test-infra: performance-tools/docs/performance-test-execution.md

* Production mode (executed periodically)
* Development mode (executed on demand)

> In the **production mode**, the script is executed periodically by `runner.sh`. This script creates a Kyma cluster that needs to be tested. Once the cluster is created, it runs all the [K6 scripts](https://github.com/kyma-project/kyma/tree/master/tests/perf) and then deletes the cluster.

Reviewer: ```suggestion
In the **production mode**, `runner.sh` periodically executes the `cluster.sh` script. This script creates a Kyma cluster that needs to be tested. Once the cluster is created, it runs all the [K6 scripts](https://github.com/kyma-project/kyma/tree/master/tests/perf) and then deletes the cluster.
```

> A kuberbetes job defined at [`performance-tests-k6-runner-cronjob.yaml`](../performance-cluster/job/performance-tests-k6-runner-cronjob.yaml), is responsible for triggering the execution of [`runner.sh`](../performance-cluster/runner.sh).

Reviewer: ```suggestion
A Kubernetes job defined in [`performance-tests-k6-runner-cronjob.yaml`](../performance-cluster/job/performance-tests-k6-runner-cronjob.yaml) triggers the execution of [`runner.sh`](../performance-cluster/runner.sh).
```

> >**NOTE:** For more details of how to write K6 scripts in Kyma refer [here](https://github.com/kyma-project/kyma/tree/master/tests/perf).

Reviewer: ```suggestion
>**NOTE:** For more details of how to write K6 scripts in Kyma, see [this document](https://github.com/kyma-project/kyma/tree/master/tests/perf/README.md).
```
